packages/matter-node.js/CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing
```

Comment:
Probably this should be moved to the root now.

Reply:
OK

Reply:
Really we should have the packages mirror the following top-level files:
```
CONTRIBUTING.md
CODE_OF_CONDUCT.md
LICENSE
```

Any unique info in this file should be moved to the README, moved to some newly named file (`BUILDING.md`?), or copied to the end of the CSA `CONTRIBUTING.md` boiler plate.

Reply:
I now copied the LICENSE and CODE_OF_CONDUCT.md also in the respective packages and added a TODO for myself to merge(sync/split CONTRIBUTING.md in matter-node.js as a follow up PR